### PR TITLE
completed prep-html-script-tag

### PR DESCRIPTION
**Pull request by mahdiehmaleki**
Screenshot

### CODE REVIEW

File: prep-html-script-tag/prep-html-script-tag-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What tag is needed to load JavaScript scripts into an HTML document?
+
+- How do you use a script tag to write JavaScript directly in the HTML document?
+
+- How do you use a script tag to load an external JavaScript file?
+
```
Comment (shawnkost) [inline prep-html-script-tag/prep-html-script-tag-quiz-notes.md]: Don't forget to answer the quiz questions!

**Review by shawnkost — CHANGES_REQUESTED** [review]
Code looks good Mahdieh, just need to answer the quiz questions!

File: prep-html-script-tag/prep-html-script-tag-quiz-notes.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# prep-html-script-tag-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What tag is needed to load JavaScript scripts into an HTML document?
```
Comment (mahdiehmaleki) [inline prep-html-script-tag/prep-html-script-tag-quiz-notes.md:7]: <script>

File: prep-html-script-tag/prep-html-script-tag-quiz-notes.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+- How do you use a script tag to write JavaScript directly in the HTML document?
```
Comment (mahdiehmaleki) [inline prep-html-script-tag/prep-html-script-tag-quiz-notes.md:9]: When the script is internal, it is placed directly between <script> and </script>.
<script>
</script>

File: prep-html-script-tag/prep-html-script-tag-quiz-notes.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+- How do you use a script tag to load an external JavaScript file?
```
Comment (mahdiehmaleki) [inline prep-html-script-tag/prep-html-script-tag-quiz-notes.md:11]: For external scripts, the src attribute specifies the URL of the external script file.
<script src="script.js"></script>

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**